README.md

```diff
@@ -26,6 +26,34 @@ generate the `.yaml` entries here.
 
 ## Using this data
 
+### Marking specific attributes
+It can be helpful to know which specific code elements of a package are vulnerable and this is done by appending an attribute and list of module paths to that attribute to the OSV payload. Eg. 
+```
+{
+  attribute: "ImageFont",
+  modules: ["PIL"]
+}
+```
+which is equivalent to `PIL:ImageFont`. If a second attribute `ImageFont2` is also affected then a second payload should be added and delimited with a `;`. Eg.
+```
+{
+  attribute: "ImageFont",
+  modules: ["PIL"];
+  attribute: "ImageFont2",
+  modules: ["PIL"]
+}
+```
```

Comment:
0.02c: I don't think we should invent custom syntax at the serialization level here. If our goal is to express the dot product, maybe both can just be arrays?

```json
{
  "attributes": ["ImageFont", "ImageFont2"],
  "modules": ["PIL"]
}
```

Reply:
(This might have negative consequences I haven't thought of yet.)

Another option would be to have this represented as an array of these objects, e.g.:

```json
[
  { "attribute": "ImageFont", "modules": ["PIL"] },
  { "attribute": "ImageFont2", "modules": ["PIL"] }
]
```

Reply:
I should have probably used a better example here, but the idea is that the two should be completely independent (code, paths, etc...) aside from the top level module. I prefer option two of your suggestions for this, but @oliverchang what do you think best fits the osv style?

Reply:
+1 to not inventing custom syntax. We should keep this as plain JSON.

```
[
  { "attribute": "ImageFont", "modules": ["PIL"] },
  { "attribute": "ImageFont2", "modules": ["PIL"] }
]
```

as @woodruffw suggested seems pretty good to me!